src/pudl/extract/ferceqr.py

```diff
@@ -0,0 +1,218 @@
+from pudl.settings import ferceqr_year_quarters
+
+logger = get_logger(f"catalystcoop.{__name__}")
+
+
+class ExtractSettings(dg.ConfigurableResource):
+    """Dagster resource which defines which EQR data to extract and configuration for raw archive."""
+
+    archive: str = "gs://archives.catalyst.coop/ferceqr/published"
+
+    @property
+    def base_path(self) -> UPath:
+        """Return UPath pointing to archive base path."""
+        return UPath(self.archive)
+
+
+@contextmanager
+def _get_csv(base_path: UPath, year_quarter: str) -> zipfile.ZipFile:
+    """Download CSV to a tempmorary directory to avoid reading into memory."""
+    year, quarter = year_quarter.split("q")
+    zip_name = f"ferceqr-{year}q{quarter}.zip"
+    remote_path = base_path / zip_name
+
+    # Create temp directory to download zip to
+    with (
+        tempfile.TemporaryDirectory() as tmp_dir,
+    ):
+        # Download file to local path
+        local_path = Path(tmp_dir) / zip_name
+        local_path.write_bytes(remote_path.read_bytes())
+        # Yield open zipfile
+        with zipfile.ZipFile(local_path) as zf:
+            yield zf
+
+
+def _clean_csv_name(csv_path: Path) -> Path:
+    """Standardize zip file names to avoid errors when opening."""
+    new_path = csv_path
+    if "'" in csv_path.name:
+        new_path = csv_path.rename(csv_path.parent / csv_path.name.replace("'", ""))
+    if '"' in csv_path.name:
+        new_path = csv_path.rename(csv_path.parent / csv_path.name.replace('"', ""))
+    return new_path
+
+
+def _get_table_name(table_type: str, year_quarter: str) -> str:
+    return f"raw_ferceqr__{table_type}_{year_quarter}"
+
+
+def _extract_ident(
+    ident_csv: str,
+    year_quarter: str,
+    duckdb_connection: DuckDBPyConnection,
+) -> str:
+    """Extract data from ident csv and write to parquet, returning CID from table."""
+    # Use duckdb to read CSV and write as parquet
+    csv_rel = duckdb_connection.read_csv(
+        ident_csv, all_varchar=True, store_rejects=True, ignore_errors=True
+    )
+    (cid,) = csv_rel.select("company_identifier").limit(1).fetchone()
+    persist_table_as_parquet(
+        csv_rel.select(f"*, '{year_quarter}' AS year_quarter"),
+        table_name=_get_table_name("ident", year_quarter),
+        partitions={"cid": cid},
+    )
+    return cid
+
+
+def _extract_other_table(
+    table_type: str,
+    csv_path: str,
+    year_quarter: str,
+    cid: str,
+    duckdb_connection: DuckDBPyConnection,
+):
+    """Extract data from ident csv and write to parquet, returning CID from table."""
+    # Use duckdb to read CSV and write as parquet
+    persist_table_as_parquet(
+        duckdb_connection.read_csv(
+            csv_path, all_varchar=True, store_rejects=True, ignore_errors=True
+        ).select(f"*, '{year_quarter}' AS year_quarter, '{cid}' as company_identifier"),
+        table_name=_get_table_name(table_type, year_quarter),
+        partitions={"cid": cid},
+    )
```

Comment:
Looking inside my `$DAGSTER_HOME` after processing several quarters, I see folder corresponding to each raw and core asset. However, none of them seem to contain the extracted data as parquet files. They each contain only a few KB of data, and the files inside have no filename extensions.

```sh
du -sh *ferceqr*
24K	core_ferceqr__contracts/
24K	core_ferceqr__identity/
24K	core_ferceqr__transactions/
24K	raw_ferceqr__contracts/
24K	raw_ferceqr__ident/
24K	raw_ferceqr__indexPub/
24K	raw_ferceqr__metadata/
24K	raw_ferceqr__transactions/
```

Looking in `$PUDL_OUTPUT/parquet/` I see directories for each of the `core_ferceqr__` tables, but those contain only the finished parquet outputs.

Where are the raw Parquet files that have been converted from the zipped CSVs?